src/lib/utils.js

```diff
@@ -248,3 +248,29 @@ export function orderLayers(activeLayersData, map, nextLayerId) {
     }
 
 }
+
```

Comment:
@KipSigei let's get some doc strings going here to describe what the function does and what it returns